harness/determined/pytorch/_experimental.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, parent: Any) -> None:
         self._data_repro_checks_disabled = False
         self._auto_to_device = True
 
-    def use_amp(self) -> None:
+    def use_amp(self, gradscaler_kwargs: Optional[Dict] = None) -> None:
```

Comment:
I'm sorry I missed this earlier.

I don't think this is an acceptable change to the public API, because `use_amp()` is supposed to be the fully automatic API, so adding kwargs to it is in opposition of the goals of the API.

Instead, for the purposes of your test, I would say you should just hack it, like this:
```python
self.context.experimental.use_amp()
# HACK: overwrite the scaler with a manually configured one, which
# is not something we don't actually allow with the use_amp() API.
self.context._scaler = amp.GradScaler(...)
self.scaler = self.context._scaler
```

Reply:
Also, in general, if you are going to make any change at all to the public api, please advertise it loudly.

Reply:
Thanks for catching this! This is done.